packages/retail-ui-extensions/src/extension-api/index.ts

```diff
@@ -1,26 +1,19 @@
-import type {CartApi} from './CartApi';
-import type {LocaleApi} from './LocaleApi';
-import type {NavigationApi} from './NavigationApi';
-import type {SessionTokenApi} from './SessionTokenApi';
+export type {LocaleApi} from './locale-api';
+export type {CartApiContent, DiscountType} from './cart-api/cart-api';
+export type {CartApi} from './cart-api';
+export type {NavigationApi, NavigationApiContent} from './navigation-api';
+export type {StandardApi} from './standard-api';
```

Comment:
Curious to know, If we are exporting `LocaleApi`(and others) through `standard-api`, why we do need to export it separately again?